azerothcore/azerothcore-wotlk
# fix(DB/Loot) battered chest loot in starting areas

## Changes Proposed:
Match battered chest loot tables in starting areas with classic.

Chests should contain:
- 1-2 water/food
- 1 grey armor piece

## Issues Addressed:
- https://github.com/azerothcore/azerothcore-wotlk/issues/9158

## SOURCE:
- https://classic.wowhead.com/object=2843/battered-chest
- https://github.com/vmangos/core/commit/b159a1cdf32417cdf3d36eaaf6389577f12fc427

## Tests Performed:
- run sql, checked ingame

## How to Test the Changes:
1. open chests in starting areas (gameobject 2843)
2. check loot https://classic.wowhead.com/object=2843/battered-chest

## Known Issues and TODO List:
- 

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
